### PR TITLE
Cli formatter unittests

### PR DESCRIPTION
**Pull request by boekhorstb1**
- added default test to see if output contains the needed info. Note: criticallity is not in cliformatter outputt
- added a test to check the color codes

Question:
Should I use loops to check for each possible log?

### CODE REVIEW

File: test/Formatter/CliFormatterTest.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        $logsarray = [$this->logMessage1, $this->logMessage2, $this->logMessage3];
+        // dd($logsarray);
+
+        foreach($logsarray as $key => $value){
+
+            $line = $f->format($value);
+
+            $loglevel = $value->level();
+            $name = $loglevel->name();
+
```
Comment (midahp) [inline test/Formatter/CliFormatterTest.php:71]: Define the color strings and use them blow to make the test more readable:
```
$RED = '\e[31m';
...
        $this->assertStringContainsString($RED, $line);  
```
Also while you are at it, you could test if every log name has the correct color.

File: test/Formatter/CliFormatterTest.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                default:
+                    $this->assertStringContainsString("\e[39m", $line);
+                    break;
+            }
+        }
+        
+
+    }
+
+
```
Comment (midahp) [inline test/Formatter/CliFormatterTest.php]: Sometimes you have unneeded empty lines in your code.
The easiest way to not worry about that is to install a `php-cs-fixer` extension for vsc.

File: test/Formatter/CliFormatterTest.php (hunk 1 above)
Comment (boekhorstb1) [reply test/Formatter/CliFormatterTest.php:71]: ok will do that !

File: test/Formatter/CliFormatterTest.php (hunk 2 above)
Comment (boekhorstb1) [reply test/Formatter/CliFormatterTest.php]: yes, will fix that. Somehow the php-cs-fixer spwans errors in my IDE

**Comment by ralflang** [comment]
Look into Horde_Cli and borrow its constants rather than duplicating them.

File: test/Formatter/CliFormatterTest.php (hunk 1 above)
Comment (ralflang) [reply test/Formatter/CliFormatterTest.php:71]: Look into Horde_Cli and borrow its constants rather than duplicating them.

Comment (boekhorstb1) [reply test/Formatter/CliFormatterTest.php:71]: > Look into Horde_Cli and borrow its constants rather than duplicating them.

ok will check

**Comment by boekhorstb1** [comment]
- Added Horde-Cli methods and constants to get colors
- checked if every log name has correct color by checking with assertEquals 
- used php-cs-fixer to cleanup code

**Review by midahp — APPROVED** [review]

**Pull request merged**